docs/api.md

```diff
@@ -62,3 +62,5 @@ Tiltfiles are written in _Starlark_, a dialect of Python. For more information o
 ---
 
 {% include api/classes.html %}
+
+<script src="/assets/js/links.js" async></script>
```

Comment:
have you considered adding this to 
https://github.com/windmilleng/tilt.build/blob/master/src/_layouts/docs.html
?

Reply:
Nope, I didn't know that existed and hadn't spent the two seconds thinking about it to realize it must exist. That seems much better, thanks!